Dockerfile

```diff
@@ -105,23 +77,39 @@ RUN apt-get install -y --no-install-recommends \
 RUN git clone https://github.com/Kurento/kms-omni-build.git /.kms \
  && cd /.kms/ \
  && git checkout 722df59b98dcdeda12151ee2d3a32c847e3fee62 \
- && git config -f .gitmodules \
-    submodule.kms-cmake-utils.commit b931efc0f5f095698956ba29f85b4aa1d784e3e0 \
- && git config -f .gitmodules \
-    submodule.kms-jsonrpc.commit 70a71812f21d5cd0cc9d4fb36c19ae48ca2f05bd \
- && git config -f .gitmodules \
-    submodule.kms-core.commit fe35efe08815926dbe511b1063cf7dbb2b91563e \
- && git config -f .gitmodules \
-    submodule.kurento-module-creator.commit 9683681dcb1bad8c5cc5d42ea313973f5857115d \
- && git config -f .gitmodules \
-    submodule.kms-elements.commit 85071d7acf47a2a87e1fa772acad2f686b7a14b4 \
- && git config -f .gitmodules \
-    submodule.kms-filters.commit cd1bab9e52864fc27a704f81dcf6ae9165ec0c78 \
- && git config -f .gitmodules \
-    submodule.kurento-media-server.commit d7c98feb60938c8b4da952363fd98da2f1f1b869 \
+ ## kms-elements fork
+ && git config -f .gitmodules submodule.kms-elements.url https://github.com/instrumentisto/kms-elements.git \
+ && git config -f .gitmodules submodule.kms-elements.branch dev \
+ ## init
  && git submodule update --init --recursive \
- && git submodule update --remote \
- && cat .gitmodules
+ ## kms-cmake-utils
+ && cd kms-cmake-utils \
+ && git checkout b931efc0f5f095698956ba29f85b4aa1d784e3e0 \
+ && cd .. \
+ ## kms-jsonrpc
+ && cd kms-jsonrpc \
+ && git checkout 70a71812f21d5cd0cc9d4fb36c19ae48ca2f05bd \
+ && cd .. \
+ ## kms-core
+ && cd kms-core \
+ && git checkout fe35efe08815926dbe511b1063cf7dbb2b91563e \
+ && cd .. \
+ ## kurento-module-creator
+ && cd kurento-module-creator \
+ && git checkout 9683681dcb1bad8c5cc5d42ea313973f5857115d \
+ && cd .. \
+ ## kms-elements
+ && cd kms-elements \
+ && git checkout 49b7fb5f600fb1a600c738cbd99a630e66b76fdf \
+ && cd .. \
+ ## kms-filters
+ && cd kms-filters \
+ && git checkout cd1bab9e52864fc27a704f81dcf6ae9165ec0c78 \
+ && cd .. \
+ ## kurento-media-server
+ && cd kurento-media-server \
+ && git checkout d7c98feb60938c8b4da952363fd98da2f1f1b869 \
+ && cd ..
```

Comment:
@alexlapa ,

Unfortunately, these changes lead to the following build error:

```
[ 82%] Building CXX object kms-elements/src/server/CMakeFiles/kmselementsimpl.dir/implementation/objects/WebRtcEndpointImpl.cpp.o
c++: internal compiler error: Killed (program cc1plus)
Please submit a full bug report,
with preprocessed source if appropriate.
See <file:///usr/share/doc/gcc-5/README.Bugs> for instructions.
make[2]: *** [kurento-media-server/server/transport/websocket/CMakeFiles/websocketTransport.dir/WebSocketTransport.cpp.o] Error 4
kurento-media-server/server/transport/websocket/CMakeFiles/websocketTransport.dir/build.make:62: recipe for target 'kurento-media-server/server/transport/websocket/CMakeFiles/websocketTransport.dir/WebSocketTransport.cpp.o' failed
make[2]: *** Waiting for unfinished jobs....
Error removing intermediate container 70a02d4e04df: No such container: 70a02d4e04dfc159b85d48397cb39aa633783c1cbe6d72d6b32e13413d5a4681
The command '/bin/sh -c mkdir -p /.kms/build/  && cd /.kms/build/  && cmake -DCMAKE_BUILD_TYPE=$TYPE           -DENABLE_ANALYZER_ASAN=ON           -DSANITIZE_ADDRESS=ON           -DSANITIZE_THREAD=ON           -DSANITIZE_LINK_STATIC=ON         ..  && make -j4' returned a non-zero code: 137
make: *** [image] Error 137
```
I think this is because the changes in the kmc-elements project were based on the Kurento version `6.9.0`, and these changes return all other sub modules of the project to version `6.8.1`.

Reply:
@flexconstructor ,

 Я вчера забилдил: `alexlapa/kurento-media-server:test`. Сейчас у меня не билдится, но по другой причине:
```
E: Failed to fetch http://security.ubuntu.com/ubuntu/pool/main/liba/libarchive/libarchive13_3.1.2-11ubuntu0.16.04.4_amd64.deb  404  Not Found [IP: 91.189.88.161 80]
```
Да и у Вас, насколько я понимаю, прична: `Killed (program cc1plus)`. Что вряд-ли связано с проблемой исходного кода.

Но по версиям согласен, надо смотреть на версию Вашего форка, и апдейтить остальные модули к этой версии.

Reply:
@flexconstructor ,

Вы `dev` в instrumentisto/kms-elements грохнули. А тут с dev'а исходники тянутся.

Reply:
@alexlapa , 
>Вы dev в instrumentisto/kms-elements грохнули. А тут с dev'а исходники тянутся.
так так и есть, но я перевел на master. Дело не в этом.

Reply:
@alexlapa ,

>  404  Not Found [IP: 91.189.88.161 80]

apt-get update решает эту проблему.

Reply:
@alexlapa , я тут наткнулся на более изящное решение построения зависимостей с помощью их [adm_scripts](https://github.com/Kurento/adm-scripts)

Reply:
@flexconstructor ,

Обновил `kms-elements`.